locustempus/main/serializers.py

```diff
@@ -36,11 +36,26 @@ class ResponseSerializer(serializers.HyperlinkedModelSerializer):
         many=True,
         view_name='api-layer-detail'
     )
+    activity = ActivitySerializer()
+
+    def create(self, validated_data):
+        try:
+            a = Activity.objects.get(
+                pk=validated_data.get('activity', None)
+            )
+
+            r = Response.objects.create(activity=a)
+            ResponseOwner(
+                response=r, owner=self.context['request'].user, activity=a)
+            return r
+        except Activity.DoesNotExist:
+            pass
```

Comment:
just my two cents...I like to have an explicit return value to make it clear what is happening.